### PR TITLE
Add /occupants indent option

### PR DESCRIPTION
**Pull request by jubalh**
Regards https://github.com/boothj5/profanity/issues/690

@kaffekanne asked for this.

### CODE REVIEW

**Comment by jubalh** [comment]
@kaffeekanne please give it a try!

**Comment by kaffeekanne** [comment]
awesome, thanks! Indenting works, but autocompletion `/occupants in<tab>` does not. `/occupants show jid` is indented 4 spaces now, i think that was ment to be 2 spaces off the nick before it was configurable. I dont need JIDs in that list, but since we have that feature, it should look correct.

Some equivalent to `/roster header char none|<char>` would complete that new behaviour would be highly appreciated. I set headers to another colour to distinguish them from nicks.

**Comment by jubalh** [comment]
> awesome, thanks! Indenting works, but autocompletion /occupants in<tab> does not.

Done now :)

> /occupants show jid is indented 4 spaces now, i think that was ment to be 2 spaces off the nick before it was configurable.

I adjusted it to be always one space more that occupants indent. I think that's enough.

**Comment by kaffeekanne** [comment]
So if the group char could be turned off this PR would be ready for merging.

File: src/ui/occupantswin.c [hunk 1]
```diff
@@ -46,18 +46,37 @@ _occuptantswin_occupant(ProfLayoutSplit *layout, Occupant *occupant, gboolean sh
     theme_item_t presence_colour = theme_main_presence_attrs(presence_str);
     wattron(layout->subwin, theme_attrs(presence_colour));
 
-    GString *msg = g_string_new("   ");
+    GString *spaces = g_string_new(" ");
+
+    int indent = prefs_get_occupants_indent();
+    int current_indent = 0;
```
Comment (paulfariello) [inline src/ui/occupantswin.c]: What is the purpose of `current_indent`?

File: src/config/theme.c [hunk 2]
```diff
@@ -452,6 +452,11 @@ _load_preferences(void)
         prefs_set_occupants_size(occupants_size);
     }
 
+    if (g_key_file_has_key(theme, "ui", "occupants.indent", NULL)) {
+        gint occupants_indent = g_key_file_get_integer(theme, "ui", "occupants.indent", NULL);
+        prefs_set_occupants_indent(occupants_indent);
```
Comment (paulfariello) [inline src/config/theme.c]: Doesn't it overwrites the user preference?

File: src/ui/occupantswin.c (hunk 1 above)
Comment (jubalh) [reply src/ui/occupantswin.c]: Right, going to remove that. I thought about passing this to win_sub_print() at first.

File: src/config/theme.c (hunk 2 above)
Comment (jubalh) [reply src/config/theme.c]: Yeah it does. But so far it always does overwrite the userprefs with the theme one if you load a theme.

**Pull request merged**